server/memphis_helper.go

```diff
@@ -474,6 +476,26 @@ func tryCreateInternalJetStreamResources(s *Server, retentionDur time.Duration,
 		INTEGRATIONS_AUDIT_LOGS_STREAM_CREATED = true
 	}
 
+	// create system tasks stream
+	if !SYSTEM_TASKS_STREAM_CREATED {
```

Comment:
This should not be created in cloud, see what we are doing with syslogs that is not created in cloud and take a reference from there